### PR TITLE
fix: resolve the issue, close #18

### LINKED ISSUE #18: Vulnerable dependency
# Bug Report

`yarn audit` finds a vulnerability in the `debug` package:

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ low           │ Regular Expression Denial of Service                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ debug                                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >= 2.6.9 < 3.0.0 || >= 3.1.0                                 │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ serverless                                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ serverless > @serverless/components >                        │
│               │ @serverless/platform-client-china > @serverless/utils-china  │
│               │ > socket.io-stream > debug                                   │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://www.npmjs.com/advisories/534                         │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

## Description

1. What did you do? 
`yarn audit`

1. What happened? 
`yarn audit` finds a vulnerability in the `debug` package.

1. What should've happened? 
`yarn audit` should not find any vulnerabilities for the `serverless` dependencies.

1. What's the content of your `serverless.yml` file? 
N/A

1. What's the output you get when you use the `SLS_DEBUG=*` environment variable (e.g. `SLS_DEBUG=* serverless deploy`) 
N/A

### PR DESCRIPTION
**Pull request by wescai**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 /node_modules
 npm-debug.log
 /package-lock.json
+.DS_Store
```
Comment (medikoo) [inline .gitignore:4]: @wescai such things should be in user gitignore file. Intentionally this ignore file is kept minimal just with rules specific to this project.

See: https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

File: package.json [hunk 2]
```diff
@@ -44,7 +44,8 @@
     "object-assign": "^4.1.1",
     "protobufjs": "^6.9.0",
     "socket.io-client": "^2.3.0",
-    "socket.io-stream": "^0.9.1",
+    "component-bind": "~1.0.0",
+    "debug": "~2.6.9",
```
Comment (medikoo) [inline package.json:48]: Let's do not introduce this dependency at all (it's a custom logger which do not use in serverless projects)

File: package.json [hunk 3]
```diff
@@ -44,7 +44,8 @@
     "object-assign": "^4.1.1",
     "protobufjs": "^6.9.0",
     "socket.io-client": "^2.3.0",
-    "socket.io-stream": "^0.9.1",
+    "component-bind": "~1.0.0",
```
Comment (medikoo) [inline package.json:47]: Let's not introduce this dependency (we can easily achieve same just by relying on native `func.bind`)

File: .gitignore (hunk 1 above)
Comment (wescai) [reply .gitignore]: done

**Pull request merged**